server/server.js

```diff
@@ -1,13 +1,26 @@
-// make this a simple http server that writes to the response stream object
 
-// the page should display the following message:
+var http = require('http');
+var enhancedDate = require('./enhancedDate');
+var trucks = require('./trucks');
 
-// Today is <day name>, <month name> <date>. Here are the available food trucks:
+var todaysDate = enhancedDate.setDate().getDate();
+var today = enhancedDate.getDayName();
+var month = enhancedDate.getMonthName();
 
-// list the food trucks returned by filterByDay, passing in the current day name.
-// e.g. filterByDay(day); where "day" is determined using the enhancedDate module
-// The list of trucks returned will be an array of food truck objects. Iterate 
-// through the list, building up a string of food truck names. Once you're done 
-// iterating through that list, display the string you built up.
 
-// Remember that the response is a stream object that must be closed.
+var getTrucks = trucks.filterByDay(enhancedDate.getDayName());
```

Comment:
You already called getDayName and assigned the result to `today` - just pass `today` in to filterByDay